src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -419,7 +414,7 @@
     </MSBuild>
     <ItemGroup>
       <VSIXSourceItem Include="@(LocalizedFilesForVsix)" />
-      <_LocalizedFiles Include="$(ArtifactsDirectory)microsoft.web.xdt\3.0.0\lib\net40\**\Microsoft.Web.XmlTransform.resources.dll" />
+      <_LocalizedFiles Include="$(ArtifactsDirectory)microsoft.web.xdt\lib\net40\**\Microsoft.Web.XmlTransform.resources.dll" />
```

Comment:
I'm assuming you validated that this still works correctly, as the path looks like a path where packages are extracted, so I find it surprising that there's no package version in the path.

Reply:
Yes the assembly is now copied from `$(PkgMicrosoft_Web_Xdt)` to a folder that does not include the version so that none of our build logic needs to worry about the version.